app/src/main/java/org/gdg/frisbee/android/widget/FeedbackFragment.java

```diff
@@ -103,6 +105,14 @@ public void onCreate(Bundle savedInstanceState) {
         addProperty("loggedIn", PrefUtils.isSignedIn(getActivity())); // Optionally add some properties
         addProperty("appStarts", PrefUtils.getAppStarts(getActivity()));
         buildProperties();
+
+        setRetainInstance(true);
```

Comment:
What about leaving the dialog open until doorbell comes back with a result?

There is also the `setErrorCallback`